### PR TITLE
Add Halloween 2022 Fanart Contest news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎨

### CODE REVIEW

**Review by tybug — CHANGES_REQUESTED** [review]

File: news/2022-10-10-halloween-fanart-contest.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Spooky month has arrived. This means one thing: time for the artists of the osu! community to demonstrate their frightening skill!
+
+![](https://assets.ppy.sh/contests/154/header.jpg)
+
+Yep, it's that time of year again. The time compression that started back in 2020 never really stopped and we're now suddenly just in October again with a burning question waiting to be answered: **do you like to draw?**
+
+If your answer to that is *yes*, then take up your implement of choice and **draw any of the osu! mascots (or community mascots) in a Halloween theme or setting** for fun and glory!
+
+You've got **10 days** to pluck some magnificently mortifying artwork out of your noggin and make it real in the medium of your choice while also following the rules listed out below. **Make sure you read them!** We've added one regarding the use of AI art generation tools since we know they're all the (fiery)rage at the moment.
```
Comment (Walavouchey) [inline news/2022-10-10-halloween-fanart-contest.md:15]: i'm sorry but i'm gonna require an extensive explanation for why fieryrage is relevant here

File: news/2022-10-10-halloween-fanart-contest.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Rules
+
+- **By submitting your artwork to this contest, you grant us permission to use your artwork in the main menu client should it win, and to use it in promotional material for both the game (osu!) and the contest (Halloween 2022 Fanart Contest).** You retain all rights to your creation and you can sell it and distribute it however you like — it is your work, after all. We're just saying here that if you submit your piece to this contest, you should be okay with us using it.
+- **All entries MUST follow the contest theme (Halloween) and contain AT LEAST one of the osu! official or community mascots.** Wouldn't be an osu! fanart contest without at least one of our beloved mascots, would it?
+- **All entries MUST be suitable for all-ages viewing.** Go easy on the gore and scare-factor. Think PG halloween, not Iron Lung levels of terminal
```
[the remark this answers is not in the document]
Comment (Walavouchey) [reply news/2022-10-10-halloween-fanart-contest.md:23]: 🤔 

```suggestion
- **All entries MUST be suitable for all-ages viewing.** Go easy on the gore and scare-factor. Think PG halloween, not Iron Lung levels of terminal [REDACTED].
```

File: news/2022-10-10-halloween-fanart-contest.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- **SUBMIT ONLY YOUR OWN WORK. REDRAWING AND COMPOSITE WORKS ARE NOT ALLOWED.** Plagiarism sucks. Don't do it. Someone in the comments will call you out for it as has happened every other time someone's tried and they're going to do more damage to your reputation than us removing your entry from the voting will do. Use only your own work — brushes and the like are fine, but that's it! Don't redraw someone else's work.
+- **AI art generation methods are not allowed.** This includes any kind of software that has you supply a written or visual prompt and fills in large amounts of details for you.
+- **Artist attribution (names, social media handles, etc) must be limited to no more than three lines total and confined to either the bottom left or bottom right corners of your entry.** Being proud of your work is good, just do it in a reasonably small spot of your work. One thing unique to osu!'s history is that we really don't like forced advertisement, and attribution that is too large can come across like it.
```
Comment (Walavouchey) [inline news/2022-10-10-halloween-fanart-contest.md:26]:
```suggestion
- **Artist attribution (names, social media handles, etc.) must be limited to no more than three lines total and confined to either the bottom left or bottom right corners of your entry.** Being proud of your work is good, just do it in a reasonably small spot of your work. One thing unique to osu!'s history is that we really don't like forced advertisement, and attribution that is too large can come across like it.
```

File: news/2022-10-10-halloween-fanart-contest.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+- **Do not leave the dotted osu! cookie template visible in your submission.** Keep the streak going. Is 2022 the year this meme finally dies?
+- **Don't use copyrighted characters from other intellectual properties.** All the official osu! mascots and the community mascots are fair game, but please avoid using representations of characters from other worlds or franchises in your work. Remember also that the community mascots are someone else's treasured creation, so please treat them with the same respect that you'd treat your own OCs and work.
+
+When you're ready to submit your piece, head on over to the [Halloween 2022 Fanart Contest listing](https://osu.ppy.sh/community/contests/154) and drag-drop your entry into the box with the big fat plus sign (\+) in it. You can delete and resubmit your entry at any time before the contest closes and voting starts. Perfect if you're like every other artist in existence who sees something minutely wrong with their work that they somehow missed in the ten hours it just took them to make the thing.
```
Comment (Walavouchey) [inline news/2022-10-10-halloween-fanart-contest.md:32]:
```suggestion
When you're ready to submit your piece, head on over to the [Halloween 2022 Fanart Contest listing](https://osu.ppy.sh/community/contests/154) and drag-drop your entry into the box with the big fat plus sign (+) in it. You can delete and resubmit your entry at any time before the contest closes and voting starts. Perfect if you're like every other artist in existence who sees something minutely wrong with their work that they somehow missed in the ten hours it just took them to make the thing.
```

File: news/2022-10-10-halloween-fanart-contest.md (hunk 1 above)
Comment (Ephemeralis) [reply news/2022-10-10-halloween-fanart-contest.md:15]: he's not, he snuck in to this because I mistakenly think I am funnier than I actually am

**Review by peppy — APPROVED** [review]

**Pull request merged**